kdl_parser/src/kdl_parser.cpp

```diff
@@ -189,6 +189,9 @@ bool treeFromXml(TiXmlDocument *xml_doc, Tree& tree)
 
 bool treeFromUrdfModel(const urdf::ModelInterface& robot_model, Tree& tree)
 {
+  if (!robot_model.getRoot())
```

Comment:
`robot_model` might be empty with undefined root. In this case access to root members below will segfault.